base/src/main/java/com/smartdevicelink/streaming/video/VideoStreamingParameters.java

```diff
@@ -216,6 +227,45 @@ public void update(VideoStreamingCapability capability, String vehicleMake) {
             this.frameRate = Math.min(this.frameRate, capability.getPreferredFPS());
         }
 
+        // This should be the last call as it will return out once a suitable format is found
+        final List<VideoStreamingFormat> formats = capability.getSupportedFormats();
+        if(formats != null && formats.size()>0){
+            for(VideoStreamingFormat format : formats){
+                for(int i = 0; i < currentlySupportedFormats.length; i ++){
+                    if(currentlySupportedFormats[i].equals(format) ){
+                        this.format = format;
+                        return;
+                    }
+                }
+            }
+            DebugTool.logWarning(TAG, "The VideoStreamingFormat has not been updated because none of the provided formats are supported.");
+
+            //TODO In the future we should set format to null, but might be a breaking change
+            // For now, format will remain whatever was set prior to this update
+        }
+
+    }
+
+    /**
+     * Update the values contained in the capability that should have been returned through the SystemCapabilityManager.
+     * This update will use the most preferred streaming format from the module.
+     * @param capability the video streaming capability returned from the SystemCapabilityManager
+     * @see com.smartdevicelink.managers.lifecycle.SystemCapabilityManager
+     * @see VideoStreamingCapability
+     */
+    @Deprecated
+    public void update(VideoStreamingCapability capability){
```

Comment:
This Deprecated method does not exist on master or develop and is not being used in this PR.

This method should be removed.